src/test/java/com/LuuQu/medicalclinic/testHelper/TestData.java

```diff
@@ -0,0 +1,311 @@
+package com.LuuQu.medicalclinic.testHelper;
+
+import com.LuuQu.medicalclinic.model.dto.*;
+import com.LuuQu.medicalclinic.model.entity.Appointment;
+import com.LuuQu.medicalclinic.model.entity.Doctor;
+import com.LuuQu.medicalclinic.model.entity.Facility;
+import com.LuuQu.medicalclinic.model.entity.Patient;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+
+public class TestData {
```

Comment:
Jak widzisz klasa nam sie troche rozrosła moze lepiej byloby wydzielic to do oddzielnych klas